.github/ISSUE_TEMPLATE/dev-ticket.md

```diff
@@ -7,35 +7,29 @@ assignees: ''
 
 ---
 
-<!--- Provide a general summary of the issue in the Title above -->
+**What**
+<!--- Describe the deliverable -->
 
-## Rationale
-<!--- Provide the relevant background information to udnerstand why this change is needed -->
-<!--- This is where you can provide more details thathelp the reader understand why you want to make this change -->
+**Why**
+<!--- Describe why the proposed changes are needed -->
 
-## Implementation details
-<!--- Explain the steps you will take to implement this change. If you do not have specific steps yet, write your implementation ideas or outline general approach -->
+**Context**
+<!--- Explain any relevant background -->
 
-## Priority
-<!--- How urgent is this issue, and why? -->
-<!--- Explain in writing or give a numerical value from 1 (nice-to-have) to 5 (critical) -->
+**Prerequisites/resources**
+<!--- does anything else need to happen to enable this work to be completed? -->
 
-## Scope
-<!--- Which parts of the project/codebase does this affect? -->
-<!--- What are the possibel downstream impacts or side-effects of this change?? -->
+**Statement of Work**
```

Comment:
I believe the title is SoW (Scope of Work?)

It should also be a checklist, this is the set of tasks that need to be completed and if the task is complex it might need to turn into a linked issue.

Also recommend adding in some predefined checklist items such as 

- [ ] if-docs updated 
- [ ] test cases added